nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/SFTPTransfer.java

```diff
@@ -110,6 +110,14 @@ private static String convertFactorySetToString(Set<String> factorySetNames) {
                 .collect(Collectors.joining(", "));
     }
 
+    private static String buildFullPath(String path, String filename) {
+        return (path == null) ? filename : (path.endsWith("/")) ? path + filename : path + "/" + filename;
+    }
+
+    private static boolean identifiesDirectory(FileAttributes attributes) {
```

Comment:
The usual convention to name methods that return boolean is to ask a question in the name usually by using is/has as the prefix.
```suggestion
    private static boolean isDirectory(FileAttributes attributes) {
```